packages/api-client/src/tcp/ReconnectingWebsocket.ts

```diff
@@ -53,14 +58,27 @@ export class ReconnectingWebsocket {
 
   private readonly logger: logdown.Logger;
   private socket?: RWS;
-
+  private pingerId?: NodeJS.Timeout;
```

Comment:
Shouldn't it be a number?
```suggestion
  private pingerId?: number;
```

Reply:
<img width="625" height="146" alt="image" src="https://github.com/user-attachments/assets/3a280e8e-f9d2-46fd-aaf2-973a8c1706bc" />

not really. this is node.js context.